common/src/main/java/com/railwayteam/railways/config/CClient.java

```diff
@@ -16,13 +18,21 @@ public class CClient extends ConfigBase {
     public final ConfigFloat trackOverlayOffset = f(0.0f, -256.0f, 256.0f, "trackOverlayOffset", Comments.trackOverlayOffset);
     public final ConfigBool useDevCape = b(true, "useDevCape", Comments.useDevCape, Comments.useDevCape2);
 
+
     // smoke
     public final ConfigGroup smoke = group(1, "smoke", Comments.smoke);
+    public final ConfigEnum<SmokeType> smokeType = e(SmokeType.OLD, "smokeType", Comments.smokeType);
```

Comment:
Shouldnt default smoke type be the new one?

Reply:
oops. yes, it should. I thought I set it to be